urlgrabber/__init__.py

```diff
@@ -52,4 +52,4 @@
               'Zdenek Pavlas <zpavlas@redhat.com>'
 __url__     = 'http://urlgrabber.baseurl.org/'
 
-from grabber import urlgrab, urlopen, urlread
+from .grabber import urlgrab, urlopen, urlread
```

Comment:
Please change all source files to consistently use Python 3 style absolute imports.

```
from __future__ import absolute_import
```

Reply:
Thanks, I'll check tha.